ignite/templates/module/create/files/base/x/{{moduleName}}/types/params.go.plush

```diff
@@ -1,62 +1,56 @@
 package types
 
 import (
-	<%= if (len(params) > 0) { %>"fmt"<% } %>
-
-	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"fmt"
 )
 
-var _ paramtypes.ParamSet = (*Params)(nil)
+var _ = fmt.Errorf
```

Comment:
Why are you referencing `fmt.Errorf` here? Are you using this line to keep "fmt" in the imports list? If so, what would happen if this line is removed?

Maybe a code comment would clarify.

Reply:
This is a bit weird. Isn't there any better way to handle this? like import `fmt` package after generating params code?

Reply:
We can create a module without a parameter, so the `fmt` import will be an `unused import`. But if we remove and add new parameters later, the `fmt` import will be missing.